### PR TITLE
feat(admin): Filament Stand Admin POC

### PR DESCRIPTION
**Pull request by AndyTWF**
- Adds the Filament Package via composer
- Adds management pages for Stand Management, including management of relations (paired stands, airlines)
- Accessible at <url>/web/admin
- Auth (for now) is done via the same method as we do for Horizon, in the future this will be Core SSO

### CODE REVIEW

File: tests/Pest.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (AxonC) [inline tests/Pest.php:1]: Is Pest being used?

File: app/Filament/Resources/StandResource.php [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                        TextInput::make('identifier')
+                            ->label(__('form.stands.identifier.label'))
+                            ->maxLength(255)
+                            ->helperText(__('form.stands.identifier.helper'))
+                            ->required()
+                            ->rule(
+                                fn (Closure $get, ?Model $record) => $get('airfield_id') ? new StandIdentifierMustBeUniqueAtAirfield(
+                                    Airfield::findOrFail($get('airfield_id')),
+                                    $record
+                                ) : function () {
```
Comment (AxonC) [inline app/Filament/Resources/StandResource.php:102]: What's occurring here with the empty closure?

File: app/Filament/Resources/StandResource/RelationManagers/PairedStandsRelationManager.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                Tables\Columns\TextColumn::make('airfield.code')
+                    ->label(__('table.stands.paired.columns.airfield'))
+                    ->sortable()
+                    ->searchable(),
+                Tables\Columns\TextColumn::make('identifier')
+                    ->label(__('table.stands.paired.columns.identifier'))
+                    ->sortable()
+                    ->searchable(),
+            ])
+            ->filters([
```
[the remark this answers is not in the document]
Comment (AndyTWF) [reply app/Filament/Resources/StandResource/RelationManagers/PairedStandsRelationManager.php]: e89f8a58

File: app/Filament/Resources/StandResource/RelationManagers/PairedStandsRelationManager.php [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+                    function (Stand $record) use ($detachAction): void {
+                        DB::transaction(function () use ($record, $detachAction) {
+                            $detachAction->getRelationship()->detach($record);
+                            $record->pairedStands()->detach($detachAction->getRelationship()->getParent());
+                        });
+                    }
+                )
+                    ->label(__('form.stands.pairs.detach.label'))
+            ])
+            ->bulkActions([
```
[the remark this answers is not in the document]
Comment (AndyTWF) [reply app/Filament/Resources/StandResource/RelationManagers/PairedStandsRelationManager.php]: e89f8a58

File: tests/Pest.php (hunk 1 above)
Comment (AndyTWF) [reply tests/Pest.php]: 9fb9ca55

File: app/Filament/Resources/StandResource.php (hunk 2 above)
Comment (AndyTWF) [reply app/Filament/Resources/StandResource.php]: Changed this to use a conditional: e54e294b

**Review by AxonC — APPROVED** [review]

**Pull request merged**